src/util.test.ts

```diff
@@ -139,13 +139,15 @@ describe('util', () => {
     });
 
     it('should timeout', () => {
+      let error;
       try {
         util.safelyExecuteWithTimeout(() => {
           return new Promise((res) => setTimeout(res, 800));
         });
       } catch (e) {
-        expect(e.message).toContain('timeout');
+        error = e;
       }
+      expect(error).toBeUndefined();
```

Comment:
This doesn't seem right :thinking: The test was certainly broken before, but it remains broken now. The test passes and continues onto the next test before the timeout resolves.

So this `expect` call makes sense - it's asserting that the error doesn't appear right away. But there needs to be a second `expect` to ensure the error is thrown eventually. Also this test block should be `async` to allow it to wait for the timeout to resolve.

Reply:
Agreed, like I mentioned in https://github.com/MetaMask/controllers/pull/425 I think it is best if I fix this in a separate PR.